addons/calendar/migrations/8.0.1.0/post-migration.py

```diff
@@ -92,11 +98,12 @@ def import_crm_meeting(cr):
             openupgrade.get_legacy_name('crm_meeting_id'),
         )
     )
-    # restore the disabled unique constraint
-    cr.execute(
-        '''alter table meeting_category_rel
-        add constraint meeting_category_rel_event_id_type_id_key
-        UNIQUE(event_id, type_id)''')
+    if constraint_enabled:
```

Comment:
Just to be sure, you tested this and in spite of the rollback, the cursor was still broken?